stakbucks/230620/b_1654_랜선자르기/app.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
+let input = fs.readFileSync(filePath).toString().split('\n');
+
+function solution(input) {
+  const [K, N] = input[0].split(' ').map(Number);
+
+  // 오영식이 갖고 있는 랜선 배열
+  const arr = [];
+  for (let i = 1; i < input.length; i++) {
+    arr.push(Number(input[i]));
+  }
```

Comment:
```suggestion
const arr = input.slice(1).map(Number);
```
요건 이렇게 축약해서 쓰면 더 편할 것 같아요

Reply:
헉 이건 처음알았네요. slice 쓸 줄은 아는데 항상 적재적소에 떠올리기가 어려운 거 같아요